panel/io/notebook.py

```diff
@@ -66,6 +70,41 @@
 }}
 """
 
+# Following JS block becomes body of the message handler callback
+bokeh_msg_handler = """
+var plot_id = "{plot_id}";
+
+if ((plot_id in window.PyViz.plot_index) && (window.PyViz.plot_index[plot_id] != null)) {{
+  var plot = window.PyViz.plot_index[plot_id];
+}} else if ((Bokeh !== undefined) && (plot_id in Bokeh.index)) {{
+  var plot = Bokeh.index[plot_id];
+}}
+
+if (plot == null) {{
+  return
+}}
+
+if (plot_id in window.PyViz.receivers) {{
+  var receiver = window.PyViz.receivers[plot_id];
+}} else if (Bokeh.protocol === undefined) {{
```

Comment:
Was this ever an issue? `Bokeh.protocol` should always be present.

Reply:
`Bokeh.protocol` was not exposed until this PR: https://github.com/bokeh/bokeh/pull/7567

You're right that this is well before the minimum supported version though and can now be removed.